Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -466,15 +466,19 @@ static CTFrameRef _DWriteGetFrame(CFAttributedStringRef string, CFRange range, C
         // Iterate through runs on the current line
         for (j = i; (j < numOfGlyphRuns) && (glyphRunDetails._baselineOriginY[j] == baselineOriginYForCurrentLine); ++j) {
             // Create _CTRun objects and make them part of _CTLine
-            _CTRun* run = [[_CTRun new] autorelease];
+            auto runRef = woc::MakeStrongCF<CTRunRef>(_CTRunCreate());
```

Comment:
or if you're gonna stick with _CTRunCreate make it return a __CTRun* so you don't need to do this cast